client/packages/requisitions/src/RnRForms/api/hooks/useRnRFormContext.ts

```diff
@@ -114,9 +114,17 @@ export const useRnRFormContext = create<RnRFormContext>((set, get) => ({
     }),
   resetSearch: () => set(state => ({ ...state, foundIds: {} })),
   search: term => {
-    let found = Object.values(get().baseLines).filter(l =>
-      itemMatchesSearch(term, l.item)
-    );
+    let numberOfMatches = 0;
+    let found = Object.values(get().baseLines).filter(l => {
+      if (numberOfMatches > 10) return false;
```

Comment:
What's the motivation for limiting? I'd understand if it was paginated, but if all the results are in the table already, I'd expect all the matching ones to highlight...

Reply:
It's optimisation, what if we clear search (and hide it) on blur too, so that highlight is only indicative while searching ?